save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/model/AgentState.kt

```diff
@@ -0,0 +1,12 @@
+package org.cqfn.save.orchestrator.model
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+enum class AgentState {
```

Comment:
These classes are now available in `save-common` in master, don't forget to remove your copies when you update this branch.